assets/components/checkboxInput/checkboxInput.jsx

```diff
@@ -0,0 +1,39 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+
+
+// ----- Types ----- //
+
+type PropTypes = {
+  id: string,
+  onChange: (preference: boolean) => void,
+  labelText?: string,
```

Comment:
I think this should be:

```jsx
labelText: string
```

Flow will apply the default prop if this field is missing.

Reply:
I prefer to leave it as `labelText?: string` to be explicit about the optional (`undefined` or a `string` but not `null`) value of this field.